docs/src/introduction.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: docs
+title:  "Introduction"
+section: "chisel3"
+---
+
+# An Introduction to Chisel
+
+_Chisel_ (Constructing
+Hardware In a Scala Embedded Language) is a hardware
+construction language embedded in the high-level programming language
+Scala.
+ Chisel is a library of special class
+definitions, predefined objects, and usage conventions within [Scala](https://www.scala-lang.org/),
+so when you write Chisel you are actually writing a Scala
+program that constructs a hardware graph.
+As you gain experience and want to make your code simpler or more
+reusable, you will find it important to leverage the underlying power
+of the Scala language. We recommend you consult one of the excellent
+Scala books to become more expert in Scala programming.
```

Comment:
Nit: the text-based formatting here is a bit wonky. Not critical, but I find either using a fixed number of lines is good or doing one-sentence-per-line (closer to what the PR does below). The latter is arguably better for version control.

Not asking for a change.

Reply:
yeah, i agree i like one-sentence-per-line / break at punctuation pattern as well, this was an artifact of me slashing out "tutorial-y" sounding code without fixing up spacing